app/views/groups/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="bg-main flex justify-between text-white items-center h-12">
+  <%= link_to "<", groups_path, class: 'text-xl mx-2'  %>
+  <h1 class="font-proxima-bold">GROUPS</h1>
+  <h2 class = 'text-xl mx-2'>></h2>
+</div>
+<%= form_with model:@group, url: groups_path, class: 'mx-auto' do |form| %>
+  <div class="relative py-8 px-5 md:px-10 bg-white shadow-md rounded border border-gray-400">
+      <h1 class="font-proxima-bold text-center text-gray-800 font-lg font-bold tracking-normal leading-tight mb-4">Add new Group</h1>
```

Comment:
- [ ] Same here, These headers suppose to be `Categories` and `Add a new category` instead of `Groups` and `Add new group`, please fix that.
- [ ] It seems that you used two `<h1>` tags here on the same page Kindly change the second one to `h2` or `h3` for better semantics.

Reply:
Thanks @Omar-Muhamad  i am changing to match requeriments.
Also changed second `h1` to `h2`